ui/src/app/components/resource-list.tsx

```diff
@@ -160,34 +154,44 @@ export const ResourcesList = fp.flow(withCdrVersions())((props: Props) => {
   };
 
   useEffect(() => {
-    const { workspaceResources } = props;
+    const { workspaces, workspaceResources } = props;
     if (workspaceResources) {
       setTableData(
-        workspaceResources.map((r) => {
-          return {
-            resource: r,
-            menu: renderResourceMenu(r),
-            resourceType: getTypeString(r),
-            resourceName: getDisplayName(r),
-            formattedLastModified: displayDateWithoutHours(
-              r.lastModifiedEpochMillis
-            ),
-            lastModifiedDateAsString: displayDate(r.lastModifiedEpochMillis),
-            cdrVersionName: getCdrVersionName(r),
-            workspaceName: getWorkspace(r)?.name,
-            lastModifiedBy: r.lastModifiedBy,
-          };
-        })
+        fp.flatMap((r) => {
+          const workspace = workspaces.find(
+            (w) => w.namespace === r.workspaceNamespace
```

Comment:
Another conversation starter, not for adjustment in this PR.

Prettier's default configuration is surprising to me in that some of the defaults favor concision and some do not (like this one). If we're going to use it, my preference would be to adjust the configuration to consistently favor concision. In this case, you would always omit parenthesis for single-argument anonymous functions. It makes a difference in marginal cases and, in my judgment, those tend to add up to significant additional context over larger areas.

Reply:
We do not currently modify `prettier`'s behavior, and I think it would take me a lot of convincing before I was inclined to start doing so.

In this case, my guess is that the rule exists to favor consistency (between handling 1 thing vs. N things) over concision.  Generally that's going to be my preference as well.